modules/installing-ove-console.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * virt/install/installing-booting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-installing-ove-console_{context}"]
+= Configuring the cluster installation from the web console
+
+Once the machines have been booted from the ISO image and the rendezvous node has been selected, configure the rest of the cluster installation from the web console.
```

Comment:
Again, this is not strictly necessary. It's perfectly legit to commence an installation by just starting the rendezvous node and jumping on the web console when available. Remaining nodes could be be booted and configured later, since in any case the UI will not let the user to move forward in the Discovery instances page, until the number of discovered hosts matches the requested cluster type (ie 3 for compact, but just 1 for SNO)

Reply:
I addressed this point in [this comment](https://github.com/openshift/openshift-docs/pull/93592/files#r2100843706), but yeah, I understand that on a technical level users could mix up the order of things a little bit. If the workflow I have documented now would work for at least 80% of users, then aside from adding notes clarifying this flexibility, I am in favor of just documenting one single workflow to keep the docs simple.